docs/release-notes/release-notes-0.18.0.md

```diff
@@ -0,0 +1,10 @@
+# Release Notes
```

Comment:
Release notes need to be rebased with the 0.18 staging branch, which has the latest template.